FeedAPIChallenge/Feed API/FeedItemsMapper.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  FeedItemsMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by Mher Movsisyan on 08.08.21.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+struct FeedItemsMapper {
+	struct Root: Decodable {
```

Comment:
`Root` is an implementation detail of `FeedItemsMapper`, so it should be marked as `private`:
```suggestion
	private struct Root: Decodable {
```

Reply:
Done